app/components/Contacts/ContactsForm.js

```diff
@@ -10,29 +10,30 @@ class ContactsForm extends React.Component {
     super(props)
 
     this.state = {
-      editing: false,
-      manualFormInput: ''
+      editing: false
     }
   }
 
   render() {
     const {
       contactsform,
+      contactsform: { showErrors },
```

Comment:
since we are declaring `contactsform` in the line above maybe this is unnecessary?

below we can just do `contactsform.showErrors`